app/src/ui/clone-repository/clone-repository.tsx

```diff
@@ -344,7 +344,7 @@ export class CloneRepository extends React.Component<
 
     if (!url) {
       const error = new Error(
-        `We couldn't find that repository. Check that you are logged in, and the URL or repository alias are spelled correctly.`
+        `We couldn't find that repository. Check that you are logged in, the network is accessible and the URL or repository alias are spelled correctly.`
```

Comment:
> ...the network is accessible<b><u>,</u></b> and the URL...

Comma please? 😄

Reply:
Instead of the word `and`? @j-f1

Reply:
Nope, just what you did just now in https://github.com/desktop/desktop/pull/2872/commits/2bd1c9cc68723205c0fbf1da9fed4dcaf8937c06.

Reply:
Pefect! I thought so. Thanks for the comment 💯